installing/installing_azure/ipi/installing-azure-customizations.adoc

```diff
@@ -40,6 +47,23 @@ include::modules/installation-configure-proxy.adoc[leveloffset=+2]
 
 * For more details about Accelerated Networking, see xref:../../../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-azure-accelerated-networking_creating-machineset-azure[Accelerated Networking for Microsoft Azure VMs].
 
+// Network Operator specific configuration
+include::modules/nw-network-config.adoc[leveloffset=+1]
```

Comment:
line spaces are needed between include statements, or dita migration breaks; I thought we have existing guidance on this, but I will double-check